stories/example-screen-form-view.stories.js

```diff
@@ -0,0 +1,451 @@
+              <span class="text-muted">Meeting Scheduled</span>
+            </div>
+          </div>
+          <div class="d-flex justify-content-right">
+            <span class="fs-6 fw-light" role="button" tabindex="0">Edit</span>
+          </div>
+        </div>
+        <div class="d-flex justify-content-between mb-2">
+          <svg width="1.125rem" height="1.125rem" class="mt-1 me-2" role="button" tabindex="0">
+            <use xlink:href="${iconSprite}#tabler-file-search" />
+          </svg>
+          <div class="d-flex flex-column w-100">
+            <div>
+              <span class="me-1">Date:</span>
+              <span class="text-muted">2019-06-14 12:03:00</span>
+            </div>
+            <div>
+              <span class="me-1">Status:</span>
+              <span class="text-muted">Meeting Scheduled</span>
+            </div>
+          </div>
+          <div class="d-flex justify-content-right">
+            <span class="fs-6 fw-light" role="button" tabindex="0">Edit</span>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+`
+
+const tabsNavigationHtml = `
+  <div class="w-100 overflow-auto p-0 mb-0 border border-light">
+    <div class="bg-white rounded-top border border-light d-flex">
+      <button class="d-flex align-items-center bg-white border-0 text-light rounded-top">
+        <svg class="icon-lg-font-size">
+          <use xlink:href="${iconSprite}#tabler-chevron-left" />
+          <span class="visually-hidden">Extend tabs' options</span>
+        </svg>
+      </button>
+      <ul class="w-100 overflow-hidden flex-nowrap nav nav-tabs nav-tabs-single-row-with-controls"></ul>
+      <button class="d-flex align-items-center bg-white border-0 text-warning rounded-top">
+        <svg class="icon-lg-font-size">
+          <use xlink:href="${iconSprite}#tabler-chevron-right" />
+          <span class="visually-hidden">Extend tabs' options</span>
+        </svg>
+      </button>
+    </div>
+  </div>
+`
+
+const form_view_html = `
+  <div class="m-0 bg-light">
+    <div class="mx-auto my-0">
+      ${LayoutLoggedHeader()}
+      <div class="d-flex mt-4 w-100">
+        <div class="d-none d-lg-block">
+          ${LayoutSidebar()}
+        </div>      
+        <div class="d-flex flex-column justify-content-start flex-wrap w-100 px-4">
+          <div class="d-flex justify-content-between align-items-center flex-wrap px-3 w-100">
+            <h2 class="m-0">00012-001-2020 (Billy Goatgruff - A1234421)</h2> 
+            <button type="button" class="btn btn-danger fs-6">
+              <svg width="1.125rem" height="1.125rem" class="mt-n1 me-1">
+                <use xlink:href="${iconSprite}#tabler-chevron-down" />
+              </svg>Other Actions
+            </button>
+          </div>
+          <div class="bg-white my-3 w-100 rounded-4">
+            ${tabsNavigationHtml}
+            <div class="row g-0">
+              <div class="col-md-8 col-sm-12 px-9 py-4">
```

Comment:
Let's change this, so that we have 20px padding around the container:

`<div class="col-md-8 col-sm-12 p-7">`

broken: 
![2021-09-08_08-34-41AM](https://user-images.githubusercontent.com/1605754/132519635-e6b58dd0-59dd-4489-a662-a7c06c90f6db.png)

fixed:
![2021-09-08_08-33-38AM](https://user-images.githubusercontent.com/1605754/132519646-8966cd88-4b65-42a3-9222-fa2a3c66a258.png)